apps/mobile/src/libs/components/wheel/libs/components/label-on-circle/label-on-circle.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+import { Circle, SvgText } from "~/libs/components/components";
+
+import { LabelCoefficient } from "../../enums/label-coefficient.enum";
+import { getCircumferencePointCoordinates } from "../../helpers/helpers";
+
+type Properties = {
+	centerPoint: number;
+	circleColor: string;
+	pointAngleRadians: number;
+	score: number;
+	text: string;
+	wheelRadius: number;
+};
+
+const ZERO = 0;
```

Comment:
```suggestion
const SVG_INITIAL_COORDINATE = 0;
```
Let's be more specific here